docs/source/notebooks/bayesian_estimation_of_ode_parameters.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>it's inital should be its initial</p><p> </p><p>Initial mispelled inital in two places</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3578/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Thanks, will get to both of these Tomorrow.

Reply:
Yeesh, there are a bunch of typos I didn't catch the first time around.